pkg/ingester/tailer.go

```diff
@@ -222,7 +221,8 @@ func (t *tailer) dropStream(stream logproto.Stream) {
 		t.blockedAt = &blockedAt
 	}
 
-	if len(t.droppedStreams) > maxDroppedStreams {
+	if len(t.droppedStreams) > t.maxDroppedStreams {
```

Comment:
We should have a test for this, if possible

Reply:
Sure. Added!